WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginListFragment.java

```diff
@@ -213,14 +215,23 @@ void onLoadedMore(boolean canLoadMore) {
             setHasStableIds(true);
         }
 
-        public void setPlugins(List<?> plugins) {
+        public void setPlugins(@Nullable List<?> plugins) {
             if (!mItems.isSameList(plugins)) {
                 mItems.clear();
-                mItems.addAll(plugins);
+                if (plugins != null) {
+                    mItems.addAll(plugins);
+                }
                 notifyDataSetChanged();
             }
         }
 
+        private void reloadPluginWithSlug(@NonNull String slug) {
```

Comment:
This is another synthetic accessor method creation.  We can make the `reloadPluginWithSlug` method protected or package-private to avoid that.